### DISCUSSION TITLE
코드 읽다가 질문입니다.

### DISCUSSION THREAD

**Discussion by ssarl2** [finko-dev/convert-published-gdocs-to-web#5] [Q&A]
개발할때 개발자들은 보통 주석을 얼마나 상세하게 다는지는 모르겠지만 정규표현식도 있고 자바스크립트의 문법이 안 익숙한 사람으로선 짧은 코드도 읽기가 힘든 것 같아요. 물론 제가 초보인 이유가 90%가 넘습니다 ㅎㅎㅎ 혹시 조금 상세한 주석 달아 주실 수 있나요? 예를 들면, [(o,n)](https://github.com/finko-dev/convert-published-gdocs-to-web/blob/main/scripts/build.js#L10) 등의 상세한 변수명이나 이 [함수](https://github.com/finko-dev/convert-published-gdocs-to-web/blob/main/scripts/build.js#L30)가 어떤 목적을 위한 것인지 등이요. 그리고 여기 이슈보드를 이렇게 써도 되나요?

**Comment by qgp9** [comment]
디스커션으로 옮겼습니다. 이슈에 올리는게 잘못된 건 아니구요.
원래는 다 이슈에 올렸었는데, 깃헙에 디스커션이 새로 생겼습니다. 일종의 일반 디스커션용 이슈트래커라고 생각해 주시면 될 것같아요.
디스커션은 그냥 게시판이나 포럼 처럼 사용하면 되고, 이슈 트래커는, 좀 더 적극적으로 무언가 수정할 필요가 있거나, 해결 할 필요가 있거나, 어쨌든 실제 contributor 들의 도움이 필요한 내용을 위해 사용하시면 됩니다.

사실 그냥 적당히 남들은 어떤 주제를 어디에 넣나 대충 눈치보시고 막 넣으시면, 누군가 이렇게 알아서 옮깁니다..

본 내용에 대한 답변은 잠시 후에!

**Reply by ssarl2** [reply]
오호, 웹 깃에 대한 기능을 안쓰다보니 몰랐는데 이렇게 알아가게 되니 재밌네요 ㅎㅎㅎ
